Automattic/jetpack
# Packages: enable autotagger for packages

#### Changes proposed in this Pull Request:

Follow-up from #19238 to enable autotagging (tagging new package releases in the mirror repos when a new release is available for that package) for more packages.

#### Jetpack product discussion

* p1617119943221100-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Check the output of the Build workflow, the new action should be present for these projects.
* Once this is merged, check that the mirror repos did not auto-tag (I already manually tagged all releases for those packages).
* When we do do a release, the mirror repos should get correctly tagged.
